Source/WebCore/crypto/keys/CryptoKeyEC.cpp

```diff
@@ -36,6 +39,19 @@ static const ASCIILiteral P256 { "P-256"_s };
 static const ASCIILiteral P384 { "P-384"_s };
 static const ASCIILiteral P521 { "P-521"_s };
 
+#if OS(DARWIN) && !PLATFORM(GTK)
```

Comment:
nit: doesn't seem necessary, since SWIFT_CPP_INTEROP is only defined for certain Darwin-based platforms.